tracer/src/Datadog.Trace/Tagging/AwsSdkTags.cs

```diff
@@ -21,7 +21,7 @@ internal abstract partial class AwsSdkTags : InstrumentationTags, IHasStatusCode
 #pragma warning disable CS0618
         [Tag(Trace.Tags.AwsRegion)]
 #pragma warning restore CS0618
-        public string AwsRegion => Region;
+        public virtual string AwsRegion => Region;
```

Comment:
Maybe this change is not needed anymore because I'm not setting it as null anymore in `AwsKinesisTags.cs`

Reply:
I was going to ask about this, it looks like it's not needed so I recommend removing the change